doc/content/documentation/configuration.md

```diff
@@ -59,6 +63,12 @@ The `loader-dev` configuration section is equivalent to the `loader` configurati
 
 This configuration entry defines a list of folders where the test files of a project can be found.
 
+### `export`
+
+To be able to use the export command you have to define these two options:
```

Comment:
Reference the `Interop` chapter to get more information about the `export` command